### ISSUE TITLE
PlantDoc: A Dataset for Visual Plant Disease Detection

### ISSUE THREAD

**Issue by Yawna** [Yawna/Convolutional-Neural-Networks#2]
PlantDoc: A Dataset for Visual Plant Disease Detection

[URL](https://dl.acm.org/doi/10.1145/3371158.3371196)

This source analyzes the data requirements of deep learning networks. Convolutional Neural Networks require large amounts of data, data that is not often available to the average farmer. PlantDoc is a dataset available to all to train artificial intelligence systems. Although this source does discuss CNNs, it may not be very helpful to others as it is rather specific to farming.

Key Points:
- CNNs in farming allows for advancements in agricultural knowledge, disease control, and yield in crops. 
- PlantDoc dataset contains 2,598 data point over 13 plant species which is fully annotated. This large set of data can increase accuracy of CNN.
- The best performing model in the study was the Faster R-CNN model. The deep learning network was built using Keras with a Tensorflow backend.

Davinder Singh, Naman Jain, Pranjali Jain, Pratik Kayal, Sudhakar Kumawat, and Nipun Batra. 2020. PlantDoc: A Dataset for Visual Plant Disease Detection. In Proceedings of the 7th ACM IKDD CoDS and 25th COMAD (CoDS COMAD 2020). Association for Computing Machinery, New York, NY, USA, 249–253. https://doi.org/10.1145/3371158.3371196
